src/utils/MastodonUtils.C

```diff
@@ -124,6 +138,30 @@ MastodonUtils::mean(const std::vector<Real> & vector)
   return sum / vector.size();
 }
 
+std::vector<Real>
+MastodonUtils::mean(const std::vector<std::vector<Real> *> & history_acc)
+{
+  // bool check = ;
+  if (MastodonUtils::checkEqualSize(history_acc) < 1)
+    mooseError("Input vectors are all not of equal size.");
+  else
+  {
+    std::vector<Real> mean_acc;
+    mean_acc.resize((*history_acc[0]).size());
+    Real tmp_var = 0;
+    for (std::size_t i = 0; i < (*history_acc[0]).size(); i++)
+    {
+      for (std::size_t j = 0; j < history_acc.size(); j++)
+      {
+        tmp_var = tmp_var + (*history_acc[j])[i];
+      }
+      mean_acc[i] = tmp_var / history_acc.size();
+      tmp_var = 0;
+    }
+    return {mean_acc};
```

Comment:
I don't think you need curly braces here.

Reply:
Removed curly braces.